Dec  1 13:24:31 node01 sshd[3149]: Accepted publickey for vagrant from 10.0.2.2 port 33726 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 13:24:31 node01 systemd[3153]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 13:24:31 node01 sshd[3149]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 13:24:31 node01 sudo[3241]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-djxccilasvbngbxhchsuyawxbajtcspy; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575206687.65-56585075862368/AnsiballZ_command.py
Dec  1 13:24:31 node01 sudo[3241]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:24:31 node01 sudo[3241]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:24:32 node01 sudo[3241]: pam_unix(sudo:session): session closed for user root
Dec  1 13:24:32 node01 sudo[3336]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-kevwzakfulqupskhqoovepkzdvxmjckq; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575206688.78-117490951365334/AnsiballZ_command.py
Dec  1 13:24:32 node01 sudo[3336]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:24:32 node01 sudo[3336]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:24:32 node01 sudo[3336]: pam_unix(sudo:session): session closed for user root
Dec  1 13:25:32 node01 sshd[3164]: Received disconnect from 10.0.2.2 port 33726:11: disconnected by user
Dec  1 13:25:32 node01 sshd[3164]: Disconnected from user vagrant 10.0.2.2 port 33726
Dec  1 13:25:32 node01 sshd[3149]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 13:25:32 node01 systemd[3158]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 13:33:27 node01 sshd[3371]: Accepted publickey for vagrant from 10.0.2.2 port 33766 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 13:33:27 node01 systemd[3375]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 13:33:27 node01 sshd[3371]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 13:33:28 node01 sudo[3463]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-tkowupfxajqkxsunsfcoozxvsshadfyi; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207224.19-143089368971618/AnsiballZ_command.py
Dec  1 13:33:28 node01 sudo[3463]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:33:28 node01 sudo[3463]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:33:28 node01 sudo[3463]: pam_unix(sudo:session): session closed for user root
Dec  1 13:33:28 node01 sudo[3558]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-fnjfolpjyzycsxgrvssrlvyubpetwbgw; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207224.81-145054513448317/AnsiballZ_command.py
Dec  1 13:33:28 node01 sudo[3558]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:33:28 node01 sudo[3558]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:33:28 node01 sudo[3558]: pam_unix(sudo:session): session closed for user root
Dec  1 13:33:44 node01 sudo[3653]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-xjrtxdknbytodgczmesjrjgghnkmpvpk; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207240.31-223170452428793/AnsiballZ_command.py
Dec  1 13:33:44 node01 sudo[3653]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:33:44 node01 sudo[3653]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:33:44 node01 sudo[3653]: pam_unix(sudo:session): session closed for user root
Dec  1 13:33:44 node01 sudo[3748]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-qhjqoxakikhknpecbnhyrmdggmjhwaae; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207240.73-156291547260143/AnsiballZ_command.py
Dec  1 13:33:44 node01 sudo[3748]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:33:44 node01 sudo[3748]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:33:44 node01 sudo[3748]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:11 node01 sudo[3843]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-qccbkcfswpahxkyirssrboldomoskbgy; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207267.39-191942228670185/AnsiballZ_setup.py
Dec  1 13:34:11 node01 sudo[3843]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:11 node01 sudo[3843]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:12 node01 sudo[3843]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:12 node01 sudo[3965]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-lwoargtregurjtltpayufjbhwdouhsih; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207269.16-90363103578872/AnsiballZ_command.py
Dec  1 13:34:12 node01 sudo[3965]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:12 node01 sudo[3965]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:12 node01 sudo[3965]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:13 node01 sudo[4060]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ybkvxyoxpwodjzbpunfgppsyawfrtbnx; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207269.54-256414516293666/AnsiballZ_command.py
Dec  1 13:34:13 node01 sudo[4060]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:13 node01 sudo[4060]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:13 node01 sudo[4060]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:35 node01 sudo[4155]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-rbnhwwiouecjgydprshtbjmkaxflbqnf; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207292.12-242161153738298/AnsiballZ_setup.py
Dec  1 13:34:35 node01 sudo[4155]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:35 node01 sudo[4155]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:36 node01 sudo[4155]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:36 node01 sudo[4277]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-wxavllnazanetfvdhakqphayaevchwhj; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207292.93-188359746704687/AnsiballZ_command.py
Dec  1 13:34:36 node01 sudo[4277]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:36 node01 sudo[4277]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:36 node01 sudo[4277]: pam_unix(sudo:session): session closed for user root
Dec  1 13:34:36 node01 sudo[4372]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-qdkjbwurlkzbodnuvavtdngabsrvdjrp; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207293.32-21822819818265/AnsiballZ_command.py
Dec  1 13:34:36 node01 sudo[4372]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:34:36 node01 sudo[4372]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:34:37 node01 sudo[4372]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:05 node01 sudo[4467]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ookejauwqtmhlfodxgwnjlcwflbwgwqi; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207321.76-96994589188671/AnsiballZ_setup.py
Dec  1 13:35:05 node01 sudo[4467]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:05 node01 sudo[4467]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:05 node01 sudo[4467]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:06 node01 sudo[4589]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-fkxbobgsbjkpaaxhkbxbqwocxrtrorri; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207322.59-13180187706946/AnsiballZ_command.py
Dec  1 13:35:06 node01 sudo[4589]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:06 node01 sudo[4589]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:06 node01 sudo[4589]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:06 node01 sudo[4684]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-hvyjhozeogcfzbiuskgefypnglynlcnv; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207322.98-188135468996989/AnsiballZ_command.py
Dec  1 13:35:06 node01 sudo[4684]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:06 node01 sudo[4684]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:06 node01 sudo[4684]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:25 node01 sudo[4779]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-srbqdgrsuazpdvjrdpwwvdvyasemkyxy; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207341.39-165711802540598/AnsiballZ_setup.py
Dec  1 13:35:25 node01 sudo[4779]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:25 node01 sudo[4779]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:25 node01 sudo[4779]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:25 node01 sudo[4901]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-cndybayuqzrpdalivrpriyquxzcdpvna; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207342.22-69496741548011/AnsiballZ_command.py
Dec  1 13:35:25 node01 sudo[4901]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:25 node01 sudo[4901]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:26 node01 sudo[4901]: pam_unix(sudo:session): session closed for user root
Dec  1 13:35:26 node01 sudo[4996]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ugzrqocrzwzqyvwnpujnurmungzlrwoz; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575207342.62-64344659058398/AnsiballZ_command.py
Dec  1 13:35:26 node01 sudo[4996]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 13:35:26 node01 sudo[4996]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 13:35:26 node01 sudo[4996]: pam_unix(sudo:session): session closed for user root
Dec  1 13:36:26 node01 sshd[3386]: Received disconnect from 10.0.2.2 port 33766:11: disconnected by user
Dec  1 13:36:26 node01 sshd[3386]: Disconnected from user vagrant 10.0.2.2 port 33766
Dec  1 13:36:26 node01 sshd[3371]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 13:36:26 node01 systemd[3380]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 14:11:15 node01 sshd[5048]: Accepted publickey for vagrant from 10.0.2.2 port 33922 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 14:11:15 node01 systemd[5052]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 14:11:15 node01 sshd[5048]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 14:11:17 node01 sudo[5087]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/bash
Dec  1 14:11:17 node01 sudo[5087]: pam_systemd(sudo-i:session): Cannot create session: Already running in a session or user slice
Dec  1 14:11:17 node01 sudo[5087]: pam_unix(sudo-i:session): session opened for user root by vagrant(uid=0)
Dec  1 14:11:19 node01 sudo[5087]: pam_unix(sudo-i:session): session closed for user root
Dec  1 14:11:21 node01 sudo[5108]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/su -
Dec  1 14:11:21 node01 sudo[5108]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:11:21 node01 sudo[5108]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:11:21 node01 su[5110]: pam_systemd(su-l:session): Cannot create session: Already running in a session or user slice
Dec  1 14:11:21 node01 su[5110]: pam_unix(su-l:session): session opened for user root by vagrant(uid=0)
Dec  1 14:11:49 node01 groupadd[5145]: group added to /etc/group: name=apache, GID=48
Dec  1 14:11:49 node01 groupadd[5145]: group added to /etc/gshadow: name=apache
Dec  1 14:11:50 node01 groupadd[5145]: new group: name=apache, GID=48
Dec  1 14:11:50 node01 useradd[5152]: new user: name=apache, UID=48, GID=48, home=/usr/share/httpd, shell=/sbin/nologin
Dec  1 14:12:52 node01 su[5110]: pam_unix(su-l:session): session closed for user root
Dec  1 14:12:52 node01 sudo[5108]: pam_unix(sudo:session): session closed for user root
Dec  1 14:12:58 node01 sshd[5063]: Received disconnect from 10.0.2.2 port 33922:11: disconnected by user
Dec  1 14:12:58 node01 sshd[5063]: Disconnected from user vagrant 10.0.2.2 port 33922
Dec  1 14:12:58 node01 sshd[5048]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 14:13:22 node01 sshd[5854]: Accepted publickey for vagrant from 10.0.2.2 port 33936 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 14:13:22 node01 systemd[5858]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 14:13:22 node01 sshd[5854]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 14:13:22 node01 sudo[5945]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-jdojycsmrnrbxkyqdcbpttmsiioxcnwg; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209618.76-245813793407243/AnsiballZ_setup.py
Dec  1 14:13:22 node01 sudo[5945]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:13:22 node01 sudo[5945]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:13:23 node01 sudo[5945]: pam_unix(sudo:session): session closed for user root
Dec  1 14:13:23 node01 sudo[6067]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-unctmrcxbewmgrdeihpykvlgiwogwaip; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209619.77-100057468543386/AnsiballZ_command.py
Dec  1 14:13:23 node01 sudo[6067]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:13:23 node01 sudo[6067]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:13:23 node01 sudo[6067]: pam_unix(sudo:session): session closed for user root
Dec  1 14:13:23 node01 sudo[6162]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-yrqlsgkrmieaazgjfeopvdibwhpqoemh; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209620.22-105775308714025/AnsiballZ_systemd.py
Dec  1 14:13:23 node01 sudo[6162]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:13:23 node01 sudo[6162]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:13:25 node01 sudo[6162]: pam_unix(sudo:session): session closed for user root
Dec  1 14:13:25 node01 sudo[6478]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-cmyapgggumteynnbapjuellsggczylhp; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209621.81-180212630199862/AnsiballZ_systemd.py
Dec  1 14:13:25 node01 sudo[6478]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:13:25 node01 sudo[6478]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:13:26 node01 sudo[6478]: pam_unix(sudo:session): session closed for user root
Dec  1 14:14:26 node01 sshd[5868]: Received disconnect from 10.0.2.2 port 33936:11: disconnected by user
Dec  1 14:14:26 node01 sshd[5868]: Disconnected from user vagrant 10.0.2.2 port 33936
Dec  1 14:14:26 node01 sshd[5854]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 14:14:26 node01 systemd[5862]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 14:14:49 node01 sshd[6726]: Accepted publickey for vagrant from 10.0.2.2 port 33940 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 14:14:49 node01 systemd[6730]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 14:14:49 node01 sshd[6726]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 14:14:49 node01 sudo[6817]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-srycxryfgbpqvnbchnvqoxnfpviloudc; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209705.81-2101164436049/AnsiballZ_setup.py
Dec  1 14:14:49 node01 sudo[6817]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:14:49 node01 sudo[6817]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:14:50 node01 sudo[6817]: pam_unix(sudo:session): session closed for user root
Dec  1 14:14:50 node01 sudo[6939]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-uuaazhjujfvktxtfnlmuetxyjsftmzua; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209706.86-153671858122390/AnsiballZ_command.py
Dec  1 14:14:50 node01 sudo[6939]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:14:50 node01 sudo[6939]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:14:50 node01 sudo[6939]: pam_unix(sudo:session): session closed for user root
Dec  1 14:14:50 node01 sudo[7034]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ypyrpkyldvdkthlotriyjwvzpyiarntr; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209707.3-35306039428724/AnsiballZ_systemd.py
Dec  1 14:14:50 node01 sudo[7034]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:14:50 node01 sudo[7034]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:14:51 node01 sudo[7034]: pam_unix(sudo:session): session closed for user root
Dec  1 14:14:51 node01 sudo[7129]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-dsahdwkkdyzlijudfdyqfscopoptwfnf; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575209707.75-56440020359901/AnsiballZ_systemd.py
Dec  1 14:14:51 node01 sudo[7129]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:14:51 node01 sudo[7129]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:14:52 node01 sudo[7129]: pam_unix(sudo:session): session closed for user root
Dec  1 14:15:52 node01 sshd[6740]: Received disconnect from 10.0.2.2 port 33940:11: disconnected by user
Dec  1 14:15:52 node01 sshd[6740]: Disconnected from user vagrant 10.0.2.2 port 33940
Dec  1 14:15:52 node01 sshd[6726]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 14:15:52 node01 systemd[6734]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 14:59:37 node01 sshd[7394]: Accepted publickey for vagrant from 10.0.2.2 port 34000 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 14:59:38 node01 systemd[7398]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 14:59:38 node01 sshd[7394]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 14:59:38 node01 sudo[7486]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-jqqweermldepufgthhssggmtkmeyqonz; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212394.5-38582223649208/AnsiballZ_setup.py
Dec  1 14:59:38 node01 sudo[7486]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:38 node01 sudo[7486]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:38 node01 sudo[7486]: pam_unix(sudo:session): session closed for user root
Dec  1 14:59:39 node01 sudo[7608]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-yckskqvzdjgsyoyrlhxssxooftwrxhef; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212395.57-91016303462818/AnsiballZ_file.py
Dec  1 14:59:39 node01 sudo[7608]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:39 node01 sudo[7608]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:39 node01 sudo[7608]: pam_unix(sudo:session): session closed for user root
Dec  1 14:59:39 node01 sudo[7702]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-tjjbpymxodkgluhgruppxontruxeeove; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212396.07-266429190490438/AnsiballZ_stat.py
Dec  1 14:59:39 node01 sudo[7702]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:39 node01 sudo[7702]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:39 node01 sudo[7702]: pam_unix(sudo:session): session closed for user root
Dec  1 14:59:57 node01 sudo[7798]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-edcgdbtmxrsvfosomqybyxyllqbvdbvx; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212414.2-225008023767476/AnsiballZ_setup.py
Dec  1 14:59:57 node01 sudo[7798]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:57 node01 sudo[7798]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:58 node01 sudo[7798]: pam_unix(sudo:session): session closed for user root
Dec  1 14:59:58 node01 sudo[7920]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-jbgcmxcsrtkbudrvbrewyyhrthwdlccs; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212415.01-275893499228761/AnsiballZ_file.py
Dec  1 14:59:58 node01 sudo[7920]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:58 node01 sudo[7920]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:58 node01 sudo[7920]: pam_unix(sudo:session): session closed for user root
Dec  1 14:59:59 node01 sudo[8014]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ztayndidrotyurpstwdhesausljayiyg; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212415.5-258047330335978/AnsiballZ_stat.py
Dec  1 14:59:59 node01 sudo[8014]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 14:59:59 node01 sudo[8014]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 14:59:59 node01 sudo[8014]: pam_unix(sudo:session): session closed for user root
Dec  1 15:00:59 node01 sshd[7409]: Received disconnect from 10.0.2.2 port 34000:11: disconnected by user
Dec  1 15:00:59 node01 sshd[7409]: Disconnected from user vagrant 10.0.2.2 port 34000
Dec  1 15:00:59 node01 sshd[7394]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 15:00:59 node01 systemd[7403]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 15:01:04 node01 sshd[8056]: Accepted publickey for vagrant from 10.0.2.2 port 34004 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 15:01:05 node01 systemd[8060]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 15:01:05 node01 sshd[8056]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 15:01:05 node01 sudo[8147]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-uhasmkprsptohqzknigmomzazoihlpys; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212481.46-176186381363327/AnsiballZ_setup.py
Dec  1 15:01:05 node01 sudo[8147]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:05 node01 sudo[8147]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:05 node01 sudo[8147]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:06 node01 sudo[8269]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-geadtltttlhukevbhlicdzqidaenffvi; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212482.57-79792848921820/AnsiballZ_file.py
Dec  1 15:01:06 node01 sudo[8269]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:06 node01 sudo[8269]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:06 node01 sudo[8269]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:06 node01 sudo[8363]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ehxccvsbuyifuinzpmiwepjonvmhipyq; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212483.03-66440067578134/AnsiballZ_stat.py
Dec  1 15:01:06 node01 sudo[8363]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:06 node01 sudo[8363]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:06 node01 sudo[8363]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:32 node01 sudo[8459]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-hjaocxregeixfhokvhexhahveogrpdha; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212508.93-232118142851892/AnsiballZ_setup.py
Dec  1 15:01:32 node01 sudo[8459]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:32 node01 sudo[8459]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:33 node01 sudo[8459]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:33 node01 sudo[8581]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-zvqvoaznixlmspctqcnzogdggfbyokab; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212509.73-272286174774562/AnsiballZ_file.py
Dec  1 15:01:33 node01 sudo[8581]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:33 node01 sudo[8581]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:33 node01 sudo[8581]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:33 node01 sudo[8675]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-fehjdbniwexrzofrzljlfzezoawhxtob; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212510.19-213573299798848/AnsiballZ_stat.py
Dec  1 15:01:33 node01 sudo[8675]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:33 node01 sudo[8675]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:33 node01 sudo[8675]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:56 node01 sudo[8771]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-nqwxmbfvydysmigxchegdxxkocpmnghb; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212532.63-265884772841573/AnsiballZ_setup.py
Dec  1 15:01:56 node01 sudo[8771]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:56 node01 sudo[8771]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:56 node01 sudo[8771]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:57 node01 sudo[8893]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-vsgwniekhxlnfypbwuvtdaholbgwbaps; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212533.48-261519700621257/AnsiballZ_file.py
Dec  1 15:01:57 node01 sudo[8893]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:57 node01 sudo[8893]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:57 node01 sudo[8893]: pam_unix(sudo:session): session closed for user root
Dec  1 15:01:57 node01 sudo[8987]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-pgipkewnnemjiqiianjcdxkvghvfumlg; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212533.92-45255602907605/AnsiballZ_stat.py
Dec  1 15:01:57 node01 sudo[8987]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:01:57 node01 sudo[8987]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:01:57 node01 sudo[8987]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:44 node01 sudo[9083]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-uqrqfoxridgsagmdkoejmsqkxbdnbkga; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212581.09-42726994621922/AnsiballZ_setup.py
Dec  1 15:02:44 node01 sudo[9083]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:44 node01 sudo[9083]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:45 node01 sudo[9083]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:45 node01 sudo[9205]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-wpamnypmytrnuamchptlpkchlofhlgdz; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212581.89-252825920649416/AnsiballZ_file.py
Dec  1 15:02:45 node01 sudo[9205]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:45 node01 sudo[9205]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:45 node01 sudo[9205]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:45 node01 sudo[9299]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-tynbgcveftqiyvvrkubbsefnvfnobupy; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212582.34-127958505559360/AnsiballZ_stat.py
Dec  1 15:02:45 node01 sudo[9299]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:46 node01 sudo[9299]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:46 node01 sudo[9299]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:55 node01 sudo[9395]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ibjhdgnzerwuupcbzkuyhkercwvloqql; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212591.91-166327935705673/AnsiballZ_setup.py
Dec  1 15:02:55 node01 sudo[9395]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:55 node01 sudo[9395]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:56 node01 sudo[9395]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:56 node01 sudo[9517]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-vlhutlrtoeeokkfqwypiqkpoakunwvtj; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212592.73-156297118571810/AnsiballZ_file.py
Dec  1 15:02:56 node01 sudo[9517]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:56 node01 sudo[9517]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:56 node01 sudo[9517]: pam_unix(sudo:session): session closed for user root
Dec  1 15:02:56 node01 sudo[9611]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-ebwqpsipnixpowfvbyitpxiugpqqucam; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212593.24-253543433369932/AnsiballZ_stat.py
Dec  1 15:02:56 node01 sudo[9611]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:02:56 node01 sudo[9611]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:02:57 node01 sudo[9611]: pam_unix(sudo:session): session closed for user root
Dec  1 15:03:57 node01 sshd[8070]: Received disconnect from 10.0.2.2 port 34004:11: disconnected by user
Dec  1 15:03:57 node01 sshd[8070]: Disconnected from user vagrant 10.0.2.2 port 34004
Dec  1 15:03:57 node01 sshd[8056]: pam_unix(sshd:session): session closed for user vagrant
Dec  1 15:03:57 node01 systemd[8064]: pam_unix(systemd-user:session): session closed for user vagrant
Dec  1 15:07:45 node01 sshd[9641]: Connection closed by authenticating user root 10.0.2.2 port 34016 [preauth]
Dec  1 15:08:08 node01 sshd[9643]: Accepted publickey for vagrant from 10.0.2.2 port 34020 ssh2: RSA SHA256:1M4RzhMyWuFS/86uPY/ce2prh/dVTHW7iD2RhpquOZA
Dec  1 15:08:08 node01 systemd[9647]: pam_unix(systemd-user:session): session opened for user vagrant by (uid=0)
Dec  1 15:08:08 node01 sshd[9643]: pam_unix(sshd:session): session opened for user vagrant by (uid=0)
Dec  1 15:08:08 node01 sudo[9734]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-bhhnrwcmgpxlvddgqcbzhykzymnwcrdp; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212904.63-271591739154915/AnsiballZ_setup.py
Dec  1 15:08:08 node01 sudo[9734]: pam_systemd(sudo:session): Cannot create session: Already running in a session or user slice
Dec  1 15:08:08 node01 sudo[9734]: pam_unix(sudo:session): session opened for user root by vagrant(uid=0)
Dec  1 15:08:09 node01 sudo[9734]: pam_unix(sudo:session): session closed for user root
Dec  1 15:08:09 node01 sudo[9856]: vagrant : TTY=pts/0 ; PWD=/home/vagrant ; USER=root ; COMMAND=/bin/sh -c echo BECOME-SUCCESS-fxhnceqflpcldwugqjejsztfyrmdalpf; /usr/bin/python /home/vagrant/.ansible/tmp/ansible-tmp-1575212905.7-44777224259365/AnsiballZ_slurp.py
